CHANGELOG.md

```diff
@@ -26,6 +26,10 @@ Newly implemented endpoints:
 | POST | /api/ga4gh/wes/v1/runs | Submit workflow |
 | GET | /api/ga4gh/wes/v1/runs/{run_id} | Workflow details |
 
+### Batch Compute Service backend removed
+
+The BCS backend and OSS filesystem (both of which support Alibaba Cloud) have been removed.
+
```

Comment:
Cromwell 80 already released [0] so this change will be in the next version.

You might get into an awkward state where Janet releases 81 before this PR merges, in which case it would become 82. Annoying, but rare and relatively easy to fix.

[0] https://github.com/broadinstitute/cromwell/releases

Reply:
In chat, talked with Janet and it sounded like we'd try to put this in the 81 release as well.